packages/nx/src/project-graph/nx-deps-cache.ts

```diff
@@ -86,8 +85,7 @@ export function createCache(
   nxJson: NxJsonConfiguration<'*' | string[]>,
   packageJsonDeps: Record<string, string>,
   projectGraph: ProjectGraph,
-  tsConfig: { compilerOptions?: { paths?: { [p: string]: any } } },
-  lockFileHash: string
```

Comment:
Should we change ProjectGraphCache to be generic type where we can add arbitrary additional fields (e.g. `lockFileHash` depending on the target setup)?